src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 3px;
+  }
+
+  &__content {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+    color: $color-main-accent;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
it's default value

Reply:
So remove default value. Because you have only several places with 500 font-weght